example/offboard_velocity/offboard_velocity.cpp

```diff
@@ -152,53 +144,39 @@ bool offb_ctrl_body(Device &device)
 
     // Turn around yaw and climb
     offboard_log(offb_mode, " Turn around yaw & climb");
-    for (unsigned i = 0; i < 200; ++i) {
-        device.offboard().set_velocity_body({0.0f, 0.0f, -1.0f, 60.0f});
-        sleep_for(milliseconds(10));
-    }
+    device.offboard().set_velocity_body({0.0f, 0.0f, -1.0f, 60.0f});
+    sleep_for(seconds(2));
 
     // Turn back
     offboard_log(offb_mode, " Turn back");
-    for (unsigned i = 0; i < 200; ++i) {
-        device.offboard().set_velocity_body({0.0f, 0.0f, 0.0f, -60.0f});
-        sleep_for(milliseconds(10));
-    }
+    device.offboard().set_velocity_body({0.0f, 0.0f, 0.0f, -60.0f});
+    sleep_for(seconds(2));
 
     // Wait for a bit
     offboard_log(offb_mode, " Wait for a bit");
-    for (unsigned i = 0; i < 200; ++i) {
-        device.offboard().set_velocity_body({0.0f, 0.0f, 0.0f, 0.0f});
-        sleep_for(milliseconds(10));
-    }
+    device.offboard().set_velocity_body({0.0f, 0.0f, 0.0f, 0.0f});
+    sleep_for(seconds(2));
     // NOTE: Use sleep_for() after each velocity-ned command to closely monitor their behaviour.
```

Comment:
Duplicate Note. If this is to stay, make sure it changes to "after velocity-body".

Reply:
Removed.